### ISSUE TITLE
Google spreadsheets are generated in personal acct

### ISSUE THREAD

**Issue by xdoan** [Sage-Bionetworks/schematic#357]
**Describe the bug**
New google spreadsheets are generated in my sagebase.org acct, not the HTANpipeline account

**To Reproduce**
Generate a template on the app and you'll see I'm the owner (or you won't be able to see bc you aren't the owner)

**Expected behavior**
HTANpipeline should be the owner

**Screenshots**
<img width="1173" alt="Screen Shot 2020-12-02 at 11 30 12 AM" src="https://user-images.githubusercontent.com/15043209/100921729-cd0cc300-3491-11eb-8203-292c822a1b32.png">
<img width="1172" alt="Screen Shot 2020-12-02 at 11 30 23 AM" src="https://user-images.githubusercontent.com/15043209/100921742-d007b380-3491-11eb-9bdf-c8428c21510a.png">

**Additional context**
I know the google api changed, but the last spreadsheets made in HTANpipeline are Nov 2020 but just for clinical templates? @milen-sage was that you testing, if so could you send me your creds and token? 
<img width="1091" alt="Screen Shot 2020-12-02 at 11 34 56 AM" src="https://user-images.githubusercontent.com/15043209/100922204-7653b900-3492-11eb-8938-f9fc5bc80f76.png">

I also noticed that the metadata_model.py example doesn't have a step to actually generate the token.pickle. When I used the gsheets quickstart for getting a token.pickle I was able to generate it with the credentials.json, but when I reran the metadata_model.py with the new token.pickle it was still in my personal gdrive so I think the creds files is also off.

**Comment by milen-sage** [comment]
Steps to resolve this issue:
- download credentials from Synapse in schematic's repo root directory, per instructions in readme
-- synapse get syn21088684
-- synapse get syn22316486
-- ensure that the credentials and quickstart just downloaded actually overwrite whatever existing ones there are

- regenerate the token pickle and authorize the application (you'd need to rename/remove your existing token.pickle for the latter)
-- when authorizing the application make sure you are logged in the HTAN pipeline google account - the best way to ensure that is to work in an incognito window
-- @sujaypatil96 could you point @xdoan as we discussed during stand up, note: she's working from the main branch

I tested the above steps just now and the spreadsheet are generated in HTAN pipeline.
@xdoan all sheets you see in your account are created by users of the HTAN data curator app.

**Comment by xdoan** [comment]
@sujaypatil96 @milen-sage so what happened was the `data_curator` instructions that have not been updated to include the quickstart file. The instructions in the `data_curator` also points to an example https://github.com/Sage-Bionetworks/schematic/blob/main/examples/metadata_model.py to generate the `token.pickle` and do not see it running the quickstart file in the code to actually generate the token. see instructions from `data_curator`: https://github.com/Sage-Bionetworks/data_curator

**Comment by xdoan** [comment]
@sujaypatil96 some thoughts for the data_curator docs, it seems too hard to maintain two schematic installation docs so maybe we should just point to schematic instructions and make that a little more robust to include installation for data_curator

**Comment by sujaypatil96** [comment]
> @sujaypatil96 @milen-sage so what happened was the data_curator instructions that have not been updated to include the quickstart file. The instructions in the data_curator also points to an example https://github.com/Sage-Bionetworks/schematic/blob/main/examples/metadata_model.py to generate the token.pickle and do not see it running the quickstart file in the code to actually generate the token. see instructions from data_curator: https://github.com/Sage-Bionetworks/data_curator

So the `metadata_model.py` file  has a line that makes a call to the `getModelManifest()` method which instantiates an object of `ManifestGenerator` class, that makes a call to `build_credentials()` which is the method that takes care of creating the `token.pickle` file, or if `build_service_account_creds()` is called then there is no need for a `token.pickle` file altogether. You're right @xdoan in that an instantiation of `MetadataModel` object does not create the `token.pickle` file and that is by design, but that can be changed. Thoughts @milen-sage?

**Comment by sujaypatil96** [comment]
> @sujaypatil96 some thoughts for the data_curator docs, it seems too hard to maintain two schematic installation docs so maybe we should just point to schematic instructions and make that a little more robust to include installation for data_curator

@xdoan: Yes, I agree. The `data_curator` docs are lacking with respect to comprehensive installation instructions. Maybe it would be a good idea to have another tab on the data ingress docs (that @ychae maintains) with comprehensive installation instructions (screenshots and what not)?

**Comment by milen-sage** [comment]
@sujaypatil96 and @xdoan let's handle the docs in a separate issue.

@xdoan my impression is that the specific google sheet ownership issue is resolved on the HTAN data curator end?

**Comment by xdoan** [comment]
@milen-sage @sujaypatil96 note to add in the step to actually generate the token.pickle in the README: `python examples/manifest_generator.py `https://github.com/Sage-Bionetworks/schematic
